libr/anal/p/anal_gb.c

```diff
@@ -714,7 +714,10 @@ static bool gb_custom_daa (RAnalEsil *esil) {
 static int gb_anop(RAnal *anal, RAnalOp *op, ut64 addr, const ut8 *data, int len, RAnalOpMask mask) {
 	int ilen = gbOpLength (gb_op[data[0]].type);
 	if (ilen > len) {
-		ilen = 0;
+		op->addr = addr;
+		op->type = R_ANAL_OP_TYPE_UNK;
+		op->size = 0;
```

Comment:
you may set op->size = ilen; this way the caller knows the size of the instruction that was not possible to disassebmle because of lack of data provided

Reply:
well, I can do that, but that requires the caller to check the return value and the value in op. ... Maybe one should consider making plugin->op return a bool instead of length?

Reply:
Yeah seems like an option here. Enforcing opsize only in one place and return bool to specify success or not. The reason can be found in the ranalop struct if needed. But that should be done in another pr. Kinda big change i guess

Reply:
yes, so for now, I'd like to keep this as it is